server/pkg/tfgrid/client.go

```diff
@@ -172,9 +192,7 @@ func (c *Client) GatewayNameDeploy(ctx context.Context, model tfgridBase.Gateway
 		return tfgridBase.GatewayNameModel{}, pkg.ErrClientNotConnected{}
 	}
 
-	projectName := generateProjectName(model.Name)
-
-	return state.cl.GatewayNameDeploy(ctx, model, projectName)
+	return state.cl.GatewayNameDeploy(ctx, model)
```

Comment:
Can you update the functions to use the new way of getting the client from state. See for example in tfchain:

```
// State from a connection. If no state is present, it is initialized
func State(conState jsonrpc.State) *TfchainState {
	raw, exists := conState[TfchainID]
	if !exists {
		ns := &TfchainState{
			client:  nil,
			network: tfchainTestnet,
		}
		conState[TfchainID] = ns
		return ns
	}
	ns, ok := raw.(*TfchainState)
	if !ok {
		// This means the invariant is violated, so panic here is ok
		panic("Invalid saved state for tfchain")
	}
	return ns
}

// Transer an amount of TFT from the loaded account to the destination.
func (c *Client) Transfer(ctx context.Context, conState jsonrpc.State, args Transfer) error {
	state := State(conState)
	if state.client == nil {
		return pkg.ErrClientNotConnected{}
	}

```